### ISSUE TITLE
No Demo Showcasing Proper use of Azure AD B2C Using Shell

### ISSUE THREAD

**Issue by markti** [Azure-Samples/active-directory-b2c-xamarin-native#86]
Ideally the demo would help developers get off the ground and running using AzureAD B2C as an authentication mechanism giving them the basic features:

1. Sign In / Sign Up Flows Integrated with Shell
2. User Context Displayed Using Shell (when logged in)
3. User Profile Page (to view user profile values when logged in)
4. Home Page to display customized welcome or prompt user to sign up / login

**Comment by charlesroddie** [comment]
I don't think this is needed as Azure AD B2C for Xamarin doesn't have a Shell dependency. It would overcomplicate the demo.

**Comment by NikolaiTsepalov** [comment]
Ideally, it should be two demos, with Shell and without, but for a single demo, I'd vote for the no Shell one.
